Update for game version 1.1.0

Pentatrate opened the pull request:
love.load() =>love.load(args)

print error to console upon crash (instead of having to open a crash log this is a quick and easy way for me)

function setModEnabled is now global (for the launcher im working on)

Pentatrate committed:
Fix for new beatblock version 1.1.0

Pentatrate committed:
Print error messages to console

Pentatrate committed:
make setModEnabled global (for the launcher)

erenkarakal commented on bbp/loader.lua:
```diff
@@ -72,7 +72,7 @@ local function getModConfigRenderer(mod)
 end
 
-local function setModEnabled(mod, enabled)
+function setModEnabled(mod, enabled)
```
```suggestion
function loader.setModEnabled(mod, enabled)
```

erenkarakal commented on lovely/mod-loader.toml:
```diff
@@ -40,12 +40,12 @@ name = "bbp.gui"
 match_indent = true
 
-# load mods near the end of love.load()
+# load mods near the end of love.load(args)
```
```suggestion
# load mods near the end of love.load
```

Pentatrate committed:
fix language loader crash (tableContains)

Pentatrate reviewed (commented):
tableContains => bbp.utils.tableContains

Pentatrate committed:
apply comments from eren
setModEnabled => loader.setModEnabled
removed params from comment for love.load

Pentatrate committed:
apply requested change to other function calls

Pentatrate reviewed (commented):
since if they both do the same thing then why did we have to make the function public even:
> bbp.loader.setModEnabled(mod, true)

> mod.enabled = true

Pentatrate reviewed (commented):
Im thinking of keeping this change regardless, i see no reason to revert

Pentatrate committed:
setModEnabled =/> loader.setMoEnabled (reverted)
